bpf/Makefile

```diff
@@ -202,8 +202,10 @@ XDP_OPTIONS = $(LB_OPTIONS) \
 	-DENABLE_NODEPORT_ACCELERATION:-DENABLE_IPV4:-DENABLE_IPV6:-DENABLE_NODEPORT:-DENABLE_DSR:-DENABLE_DSR_HYBRID: \
 	-DENABLE_NODEPORT_ACCELERATION:-DENABLE_IPV4:-DENABLE_IPV6:-DENABLE_NODEPORT:-DENABLE_DSR:-DDSR_ENCAP_MODE:-DDSR_ENCAP_NONE:-DDSR_ENCAP_IPIP=2 \
 	-DENABLE_NODEPORT_ACCELERATION:-DENABLE_IPV4:-DENABLE_IPV6:-DENABLE_NODEPORT:-DENABLE_DSR:-DDSR_ENCAP_MODE:-DDSR_ENCAP_IPIP:-DDSR_ENCAP_NONE=2 \
+	-DENABLE_NODEPORT_ACCELERATION:-DENABLE_IPV4:-DENABLE_IPV6:-DENABLE_NODEPORT:-DENABLE_DSR:-DDSR_ENCAP_MODE:-DDSR_ENCAP_GENEVE:-DDSR_ENCAP_IPIP=2 \
```

Comment:
We should also add some additional configs in `bpf/complexity-tests`. But imho it's ok to add that as follow-on.

Reply:
We should set `bpf/complexity-tests` to maximize the number of instructions, right? So should we add `-DDSR_ENCAP_MODE=1 -DDSR_ENCAP_GENEVE=1 -DDSR_ENCAP_IPIP=2` to all configs? Will the program size be larger in GENEVE than in IP option mode?

Reply:
I have added the complexity test configs for DSR with Geneve. Does it meet your expectation?
https://github.com/cilium/cilium/pull/23890/commits/8e6d13ec816b2e553f89172578db8b246cd04ade

Reply:
looks good, thank you!